Description: Do not add an rpath when installing to multiarch path.
Author: Andreas Metzler <ametzler@debian.org>
Origin: vendor
Last-Update: 2013-07-07

--- autogen-5.17.4.orig/autoopts/mk-autoopts-pc.in
+++ autogen-5.17.4/autoopts/mk-autoopts-pc.in
@@ -31,7 +31,7 @@
       cflags="-I${includedir}"
 test 'X@ENABLE_STATIC@' = Xno && static_libs=''
 case "${libdir}" in
-/lib | /lib64 | /usr/lib | /usr/lib64 )
+/lib | /lib64 | /usr/lib | /usr/lib64 | /usr/lib/* )
     ldopts=''
     ldflags=-lopts
     ;;
--- autogen-5.17.4.orig/autoopts/autoopts-config.in
+++ autogen-5.17.4/autoopts/autoopts-config.in
@@ -31,7 +31,7 @@
       cflags="-I${includedir}"
 test 'X@ENABLE_STATIC@' = Xno && static_libs=''
 case "${libdir}" in
-/lib | /lib64 | /usr/lib | /usr/lib64 )
+/lib | /lib64 | /usr/lib | /usr/lib64 | /usr/lib/* )
     ldopts=''
     ldflags=-lopts
     ;;
